tests/gradients/test_gradient_transform.py

```diff
@@ -142,17 +142,53 @@ def test_multiple_tensor_arguments(self, tol):
 
         @qml.qnode(dev)
         def circuit(x, y):
-            qml.RX(x[0, 0], wires=0)
-            qml.RY(y[0, 0], wires=0)
-            qml.RZ(x[1, 0], wires=0)
+            qml.RX(x[0], wires=0)
+            qml.RY(y[0], wires=0)
+            qml.RZ(x[1], wires=0)
             return qml.probs(wires=[0, 1])
 
-        x = np.array([[0.1], [0.2]], requires_grad=True)
-        y = np.array([[0.2], [0.3]], requires_grad=True)
+        x = np.array([0.1, 0.2], requires_grad=True)
+        y = np.array([0.2, 0.3], requires_grad=True)

```

Comment:
why was this test modified? Shouldn't that original use-case still work?

Reply:
Due to the different axes ordering in the output (this is one of the test cases that were stacked and transposed), `qml.math.squeeze` has a different effect here (actually do something because the last dimension is the last dimension of the two QNode arguments, `1`) than it did before (not doing anything because the last dimension was the output dimenstion of the QNode, `4`).

Reply:
Makes sense!